### DISCUSSION TITLE
Running into module errors when installing tailwinds

### DISCUSSION THREAD

**Discussion by xn1cklas** [remix-run/remix#7572] [Q&A]
First time starting a remix project. Wanted to use tailwindcss with remix and ran into this project.

I followed this guide for installation: https://tailwindcss.com/docs/guides/remix

```zsh
✘ [ERROR] Build failed with 1 error:
app/root.tsx:11:19: ERROR: [plugin: css-file] module is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and '/Users/my-remix-app/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension. [plugin css-bundle-plugin]

    app/root.tsx:20:30:
      20 │ import { cssBundleHref } from "@remix-run/css-bundle";
         ╵                               ~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] module is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and '/Users/my-remix-app/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension. [plugin css-file]

    app/root.tsx:22:19:
      22 │ import styles from "./tailwind.css";
         ╵                    ~~~~~~~~~~~~~~~~
```

**Comment by seamus-mcd** [comment]
Try making sure you use "export default {config}" instead of "module.exports = {config}" for every config file in the root of your project

**Comment by tyler-dot-earth** [comment]
@nsquaredotone did you figure this out? I'm having a similar issue.

**Reply by tyler-dot-earth** [reply]
Hold up, I think I figured it out! My remix v1 blues stack had a compiled tailwind file at `app/styles/tailwind.css`, but newer Blues Stack has a not-compiled tailwind file at `app/tailwind.css` with the contents:
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```
and then my `app/root.tsx` file needed to have its css import update to point at `app/tailwind.css` (NOT :x:`app/styles/tailwind.css`:x:)
